src/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -2535,7 +2535,7 @@ public static void RemoveProviderTransparent(TypeDescriptionProvider provider, T
         /// associated with.
         /// 
         /// This method can be called from partially trusted code. If 
-        /// <see cref="TypeDescriptorPermissionFlags.RestrictedRegistrationAccess"/>
+        /// <see cref="E:System.Security.Permissions.TypeDescriptorPermissionFlags.RestrictedRegistrationAccess"/>
```

Comment:
📝 I previously assumed that `E:` was added here because this assembly doesn't reference an assembly defining the `TypeDescriptorPermissionFlags.RestrictedRegistrationAccess` member. If this is not correct, the preferable form for this reference and the one above it would be removing the `E:` prefix, since the inclusion of a prefix causes the compiler to not validate the reference at all.

Reply:
There is no reference so without the `E:` it throws an error even with the fully qualified name.